src/main/java/lebron/task/Deadline.java

```diff
@@ -0,0 +1,35 @@
+package lebron.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This class represents the Deadline event.
+ *
+ * @author Nigel Tan
+ */
+public class Deadline extends Task {
+
+    private LocalDate by;
+
+    /**
+     * Constructor.
+     *
+     * @param description the name of the task
+     * @param by the deadline
+     */
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = LocalDate.parse(by);
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Shouldn't there be a javadoc for this toString method? Not sure if you have left it out purposely or not!